imqsauth/http.go

```diff
@@ -1115,72 +1105,26 @@ func getUserIdOrIdentity(r *httpRequest) (string, authaus.UserId, error) {
 	}
 }
 
-func (central *ImqsCentral) handleWebSocket(ws *websocket.Conn) {
-	closeConnection := true
-	var token authaus.Token
-	if token, err := authaus.HttpHandlerPrelude(&central.Config.Authaus.HTTP, central.Central, ws.Request()); err == nil {
-		if permList, errDecodePerms := authaus.PermitResolveToList(token.Permit.Roles, central.Central.GetRoleGroupDB()); errDecodePerms == nil {
-			closeConnection = !permList.Has(PermEnabled)
-		}
-	}
-
-	if closeConnection {
-		ws.Close()
-		return
+func (central *ImqsCentral) broadcastToAllSubscribers(msg string) {
+	notificationReqJson := &notificationRequestJson{}
+	notificationReqJson.Channel = "authNotifications"
+	notificationReqJson.Msg = msg
+	jsonString, err := json.Marshal(notificationReqJson)
```

Comment:
You can change the json.Marshal(notificationReqJson) into json.Marshal(&notificationRequestJson{"authNotifications", msg}) if you want to make it simpler